backend/routes/event/join_event.js

```diff
@@ -0,0 +1,31 @@
+const { pool } = require('../../lib/database')
+const { validDate, validTime } = require('../../lib/validation')
+const { timeForSQL, dateForSQL } = require('../../lib/formatation')
+const { InputValidationError } = require('../../lib/error')
+
+async function create(userId, locationId, eventTime, eventDate) {
+  await pool.execute('INSERT INTO join_up_at (user_id, location_id, event_time, event_date) ' +
+    'VALUES (?, ?, ?, ?)', [userId, locationId, eventTime, eventDate])
+}
+
+// TODO: error for invalid inputs
+async function createJoinEvent(req, res) {
+  const { userId } = req.token
+  const { locationId } = req.body
+  let { eventTime, eventDate } = req.body
+  if (!validDate(eventDate)) {
+    throw new InputValidationError('eventDate', 'Date must be of type date.')
+  }
+  if (!validTime(eventTime)) {
+    throw new InputValidationError('eventDate', 'Date must be of type time.')
```

Comment:
Copy paste mistake. Property of input validation error is wrong. Should be eventTime